CakePHP/Tests/Commenting/TypeHintUnitTest.1.inc

```diff
@@ -36,4 +36,7 @@ function test()
 
     /** @psalm-var null|scalar|class-string<\Cake\I18n\Number>|array $testVar2 */
     $testVar2;
+
+    /** @var string|null|array<string|int, array<mixed, array<string>|string|null>|string|null>
```

Comment:
This seems to be missing closing tag though

Reply:
It's impossible to read, but after breaking it down manually, it should be right.

Reply:
I mean the closing tag of */
For the comment to be valid docblock

Reply:
oh yea! good catch!